src/Microsoft.AspNet.Mvc.ModelBinding/Validation/DefaultModelValidatorProvider.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNet.Mvc.ModelBinding.Validation
+{
+    public class DefaultModelValidatorProvider : IModelValidatorProvider
```

Comment:
how many validators do we have in the system?  do we really need more than `IValidatorProvider` (or `IValidationProvider`) and `DefaultModelValidatorProvider` (do the or yourself)?